package.json

```diff
@@ -40,14 +40,13 @@
     "*.js": "npm run lint:fix"
   },
   "dependencies": {
-    "@skyscanner/eslint-config-skyscanner": "^15.0.0",
-    "@skyscanner/stylelint-plugin-backpack": "^2.0.1",
-    "stylelint": "^14.16.1",
-    "stylelint-config-prettier": "^9.0.4",
-    "stylelint-config-standard-scss": "^3.0.0",
+    "@skyscanner/eslint-config-skyscanner": "^15.2.0-beta.0",
+    "@skyscanner/stylelint-plugin-backpack": "^3.0.0-beta.1",
+    "stylelint": "^15.10.2",
+    "stylelint-config-standard-scss": "^10.0.0",
```

Comment:
Stylelint v15 recommends extending `stylelint-config-standard`. We do this via `stylelint-config-standard-scss`, which wraps it, so we bump it to latest to match.

https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/681c734d0139d004f51cf261d48e74c5bb645d08/index.js#L4

https://github.com/stylelint-scss/stylelint-config-standard-scss/releases